frontend/kesaseteli/employer/src/hooks/application/useGetApplicationFormFieldLabel.ts

```diff
@@ -0,0 +1,12 @@
+import { getApplicationFormFieldLabel } from 'kesaseteli/employer/utils/application.utils';
+import { useTranslation } from 'next-i18next';
+import type ApplicationFieldName from 'shared/types/application-field-name';
+
+const useGetApplicationFormFieldLabel = (
+  field: ApplicationFieldName
+): string => {
+  const { t } = useTranslation();
+  return getApplicationFormFieldLabel(t, field);
```

Comment:
Maybe in both services, we may omit the `ApplicationForm` part, to get shorter variable names.

```suggestion
  return getFieldLabel(t, field);
```

Reply:
This hook has some changes in upcoming pull-requests so I would not rename the file right now. Good idea otherwise.